tests/baselines/reference/api/typescript.d.ts

```diff
@@ -5445,6 +5445,19 @@ declare namespace ts {
     interface RegularExpressionLiteral extends LiteralExpression {
         readonly kind: SyntaxKind.RegularExpressionLiteral;
     }
+    enum RegularExpressionFlags {
```

Comment:
This enum is unused, which implies it shouldn't be public.

Reply:
That's because originally I actually intended to add a few properties to `RegularExpressionLiteral`:
```ts
export interface RegularExpressionLiteral extends LiteralExpression {
    readonly kind: SyntaxKind.RegularExpressionLiteral;
    readonly regExpBody: string;
    readonly regExpFlags: RegularExpressionFlags;
    readonly regExpFlagsText: string;
}
```
I guess I'll defer the enum exposure to the next PR about the follow-up proposal then. I will mark the enum internal later, before the PR is ready to ship.